src/AutorestSwift/Code Generator/Manager.swift

```diff
@@ -46,7 +46,7 @@ class Manager {
     init(withInputUrl input: URL, destinationUrl dest: URL) {
         self.inputUrl = input
         // TODO: Make this configurable
-        destinationRootUrl = dest.appendingPathComponent("generated").appendingPathComponent("sdk")
+        self.destinationRootUrl = dest.appendingPathComponent("generated").appendingPathComponent("sdk")
```

Comment:
Is the being changed by swiftlint/swiftformat? Seems like it keeps bouncing back and forth.

Reply:
this is change by swiftlint